ComparePaymentProcessors.md

```diff
@@ -0,0 +1,95 @@
+# Payment Processor Comparison
+
```

Comment:
I think this can be added as an introduction https://github.com/btcpayserver/btcpayserver-doc#how-is-it-different